index.html

```diff
@@ -437,11 +437,11 @@ <h2>
         Navigation scope
       </h2>
       <p data-link-for="WebAppManifest">
-        A <dfn data-export="">navigation scope</dfn> is a <a>URL</a> that
-        represents the set of URLs to which an <a>application context</a> can
-        be navigated while the manifest is <a>applied</a>. The <a>navigation
-        scope</a> of a manifest <var>manifest</var> is
-        <var>manifest</var>["<a>scope</a>"].
+        The {{WebAppManifest/scope}} member of a [=processed manifest=] is the
```

Comment:
Nit: I think I'd prefer this to be phrased around the other way to avoid it looking like Yoda wrote it :)

"The **navigation scope** of an application context is the `scope` member of the processed manifest that is applied to it."

(As a bonus: we link this back up to the concept of "applied".)

Reply:
Actually, just realised that this changes "navigation scope" to be a property of the application context, not the manifest. I think that's a bit backwards, because then you have this twisted structure:

- The `scope` is a property of the processed manifest.
- The "navigation scope" is a property of the application context (referring to the processed manifest applied to the application context).
- A URL can be "within scope" of a URL.
- A URL can be "within scope" of a manifest (referring to the navigation scope of the application context the manifest is applied to).
- Lots of definitions are concerned with whether a URL is "within scope of an application context's manifest" (referring to the processed manifest applied to the application context).

So if something says "if a URL is within scope of the application context's manifest", expanding out all of these definitions require that we:

1. Start with an application context,
2. Get the manifest applied to it (argument to "within scope of a manifest"),
3. Get the application context it is applied to (argument to "navigation scope"),
4. Get the manifest applied to it (to get the `scope` property),
5. Check whether the `scope` is a prefix of URL.

We should be able to cut out steps 3 and 4. I think this can be achieved by navigation scope continuing to be a property of a manifest, not an application context.

We could also define "navigation scope of an application context" and "within scope of an application context", but I think that's overkill.

Reply:
> applied to it.

I wanted to avoid "it", as "it" is ambiguous. Will try to rephrase... 

> We should be able to cut out steps 3 and 4. I think this can be achieved by navigation scope continuing to be a property of a manifest, not an application context.

Yes, that could work... although an application context is formed by the applied processed manifest.  

> We could also define "navigation scope of an application context" and "within scope of an application context", but I think that's overkill.

Agree.

Reply:
Ok, @mgiuca do you agree? For simplicity: 

```
        The <dfn>navigation scope</dfn> is the {{WebAppManifest/scope}} member of a [=processed
        manifest=]. 

        The navigation scope restricts the set of URLs to which an
        [=application context=] can be [=navigated=] while the manifest is
        [=applied=].
```